pkg/controllers/provisioningv2/rke2/machineprovision/args.go

```diff
@@ -118,7 +128,12 @@ func (h *handler) getArgsEnvAndStatus(infraObj *infraObject, args map[string]int
 		cmd = append(cmd, "create",
 			fmt.Sprintf("--driver=%s", driver),
 			fmt.Sprintf("--custom-install-script=/run/secrets/machine/value"))
-
+		if rke2.WindowsCheck(driver) {
+			machineOS, ok := supportedMachineOS[bootstrapOS]
```

Comment:
I don't see anywhere where `bootstrapOS` is getting set anywhere, so I think it will always be an empty string here.

Reply:
I think this might be why the provisioning tests are failing.

Reply:
Fixed, thanks for catching this!